languages/common-lisp/exercises/concept/basics/.docs/after.md

```diff
@@ -0,0 +1,9 @@
+This exercise's goal was to begin learning about Common Lisp with a small
```

Comment:
We've recently added a function to the `after.md` document, in that it will also become a reference for students to later refer back to. As such, we'd like all syntax introduced by the exercise to also be listed in the `after.md`. See an example [here](https://github.com/exercism/v3/blob/fsharp/strings/after-syntax/languages/fsharp/exercises/concept/strings/.docs/after.md).